cmd/system-probe/modules/tcp_queue_tracer.go

```diff
@@ -27,7 +27,7 @@ var TCPQueueLength = module.Factory{
 	Name:             config.TCPQueueLengthTracerModule,
 	ConfigNamespaces: []string{},
 	Fn: func(cfg *config.Config) (module.Module, error) {
-		t, err := probe.NewTCPQueueLengthTracer(ebpf.NewConfig())
+		t, err := tcpqueuelength.NewTCPQueueLengthTracer(ebpf.NewConfig())
```

Comment:
As the name of the check (`tcpqueuelength`) is now part of the package name, would it make sense to simplify the name of the function ?

```suggestion
		t, err := tcpqueuelength.NewTracer(ebpf.NewConfig())
```